### PR TITLE
Cleanup unused content.

### PR DESCRIPTION
**Pull request by moonheart08**
Removes some unused content.

### CODE REVIEW

**Comment by Elijahrane** [comment]
looks used to me

**Comment by EmoGarbage404** [comment]
!1984

**Comment by Emisse** [comment]
One of the seven angels who had the seven bowls full of the seven last plagues came and said to me, "Come, I will show you the bride, the wife of the Lamb." And he carried me away in the Spirit to a mountain great and high, and showed me the sexy clown, coming down out of heaven from God. It shone with the glory of God, and its brilliance was like that of a very precious jewel, like a jasper, clear as crystal.

**Comment by GoodWheatley** [comment]
Merged sexy clown and then removes it 2 seconds later, absolute chad

File: Resources/Prototypes/Entities/Clothing/Masks/masks.yml [hunk 1]
```diff
@@ -143,18 +143,6 @@
...
-  parent: ClothingMaskBase
-  id: ClothingMaskSexyClown
-  name: sexy-clown mask and wig
-  description: A feminine clown mask for the best clowns.
-  components:
-  - type: Sprite
-    sprite: Clothing/Mask/sexyclown.rsi
-  - type: Clothing
-    sprite: Clothing/Mask/sexyclown.rsi
-  - type: BreathMask
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Masks/masks.yml:156]: this needs identity blocker

**Comment by 5150Con** [comment]
I see the path that has been chosen by the Lizard Den, and I weep for the generations to come. No longer is the server a beacon of peace and progress, but a den of deceit and regression set on tainting the future for all. Sexy Clown was the last outcry of an advanced empire, before it collapsed due to interior strife and stagnation. There does not exist a word to describe the bleakness that is upon us, yet words come upon me in waves of inner turmoil, raging to be set free and right this terrible wrong that has been done to all.

**Pull request merged**